stories/molecules/SelectBadge.stories.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Option, Priority, SelectBadge } from '@textkernel/oneui';
+
+const meta: Meta<typeof SelectBadge> = {
+    title: 'Molecules/SelectBadge',
+    component: SelectBadge,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof SelectBadge>;
+
+const commonArgs = {
+    children: 'London',
+    option: { value: '15', label: '+15 km' },
+    optionList: [
+        { value: '5', label: '+5 km' },
+        { value: '15', label: '+15 km' },
+        { value: '25', label: '+25 km' },
+    ],
+    optionListHeader: 'Radius',
+};
+
+export const _SelectBadge: Story = {
```

Comment:
I don't see any actions logged in the console when priority changes and so on. Please log actions so that it is visible as in other stories.